skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt

```diff
@@ -143,7 +155,7 @@ actual open class SkiaLayer internal constructor(
         addPropertyChangeListener("graphicsContextScaleTransform") {
             Logger.debug { "graphicsContextScaleTransform changed for $this" }
             latestReceivedGraphicsContextScaleTransform = it.newValue as AffineTransform
-            redrawer?.syncSize()
+            revalidate()
```

Comment:
Changed, because layout depends on the scale inside `adjustSizeToContentScale`

Reply:
Probably it can fix https://youtrack.jetbrains.com/issue/CMP-5811/Moving-Desktop-window-across-monitors-causes-change-in-layout-since-1.7.0-alpha01-when-running-from-the-IDE

Reply:
Sounds like it could bring back https://youtrack.jetbrains.com/issue/CMP-6550/Resizing-the-window-using-WindowState-breaks-the-UI

Is there a pure Compose reproducer I can try out?

Reply:
A pure Compose is:
```
import androidx.compose.foundation.background
import androidx.compose.foundation.layout.Box
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.ui.Modifier
import androidx.compose.ui.awt.ComposePanel
import androidx.compose.ui.graphics.Color
import kotlinx.coroutines.delay
import kotlinx.coroutines.runBlocking
import org.jetbrains.skiko.MainUIDispatcher
import java.awt.Dimension
import javax.swing.JFrame
import javax.swing.JLayeredPane
import javax.swing.WindowConstants

fun main() {
    runBlocking(MainUIDispatcher) {
        val window = JFrame()
        val panel = ComposePanel()
        panel.setContent {
            Box(Modifier.fillMaxSize().background(Color.Red))
        }

        panel.size = Dimension(100, 100)
        val box = javax.swing.Box.createVerticalBox().apply {
            add(panel)
        }
        box.setBounds(0, 0, 100, 100)

        try {
            val container = JLayeredPane()
            container.add(box)
            window.contentPane.add(container)
            window.setLocation(200, 200)
            window.size = Dimension(200, 200)
            window.defaultCloseOperation = WindowConstants.DISPOSE_ON_CLOSE
            window.isUndecorated = true
            window.isVisible = true

            delay(1000)

            box.setBounds(100, 0, 100, 100)
            delay(10000)
        } finally {
            window.dispose()
        }
    }
}
```

After

Reply:
I checked https://youtrack.jetbrains.com/issue/CMP-6550/Resizing-the-window-using-WindowState-breaks-the-UI with this Skiko, it doesn't reproduce (it reproduces in 1.7.0-beta02 on my machine)